### PR TITLE
Add Appendix A as contact information

### LINKED ISSUE #163: Add Appendix A - Event and FTC Info/Contact Page
We can discuss where this need needs to go. Maybe a link in the footer?

### PR DESCRIPTION
**Pull request by uvidyadharan**
Resolves #163

### CODE REVIEW

File: docs/source/contact/contact.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+   `FIRST Tech Challenge Event
+   Schedule <http://www.firstinspires.org/team-event-search>`__ – Find
+   *FIRST* Tech Challenge events in your area.
+
+FIRST Tech Challenge Social Media
+---------------------------------
+
+   `FIRST Tech Challenge Twitter Feed <https://twitter.com/FTCTeams>`__
+   - If you are on Twitter, follow the *FIRST* Tech Challenge Twitter
+   feed for news updates.
```
Comment (cmacfarl) [inline docs/source/contact/contact.rst:66]: Twitter/Facebook/YouTube should all use the standard row of icons to represent the fact that the organization participates in social media.  It is not at all necessary to suggest to people that they "follow" any particular social media account.  Everyone knows how these things work these days.  We should let them choose for themselves whether or not they want to follow our accounts.

File: docs/source/contact/contact.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+   Contains training videos, game animations, news clips, and more.
+
+   `FIRST Tech Challenge
+   Blog <http://firsttechchallenge.blogspot.com/>`__ – Weekly articles
+   for the *FIRST* Tech Challenge community, including outstanding
+   volunteer recognition!
+
+   `FIRST Tech Challenge Team Email
+   Blasts <http://www.firstinspires.org/node/4311>`__ – contain the most
+   recent *FIRST* Tech Challenge news for teams.
```
Comment (cmacfarl) [inline docs/source/contact/contact.rst:83]: The blog and newsletter are consumption resources, not contact resources.  They'd be better referenced in a media sidebar category?  Would that media sidebar perhaps also contain the Twitter/Faceback/YouTube references?

**Review by cmacfarl — CHANGES_REQUESTED** [review]

File: docs/source/contact/contact.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
+Contact Us
+======================
+
+:doc:`Game Forum Q&A </game_specific_resources/ftcqa/ftcqa>`
```
[the remark this answers is not in the document]
Comment (texasdiaz) [reply docs/source/contact/contact.rst:4]: Normally I would agree, but this "Contact Us" page is a compilation of all the ways a team can Contact FIRST and ways that FIRST communicates to teams. I agree that we should not merely link to the Q&A, but we should link to our internal Q&A page here:
http://ftc-docs.firstinspires.org/game_specific_resources/ftcqa/ftcqa.html

File: docs/source/contact/contact.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+   *FIRST* Tech Challenge.
+
+   `FIRST Tech Challenge Volunteer
+   Resources <http://www.firstinspires.org/node/5146>`__ – To access
+   public volunteer manuals.
+
+   `FIRST Tech Challenge Event
+   Schedule <http://www.firstinspires.org/team-event-search>`__ – Find
+   *FIRST* Tech Challenge events in your area.
+
```
[the remark this answers is not in the document]
Comment (texasdiaz) [reply docs/source/contact/contact.rst:60]: > None of this is contact information. As such it doesn't belong on a contact page. 
> The "for everything FTC" url should go on the About page.

I agree.

> We don't have a volunteer section on ftc-docs yet. When we do that would be the 
> appropriate place for the volunteer related URL.

Agreed.

> Events search should not link to the team-event-search. Event search should 
> _always_ link to ftc-events. Team search should go on the FAQ page.

Agreed.

File: docs/source/contact/contact.rst (hunk 2 above)
Comment (texasdiaz) [reply docs/source/contact/contact.rst:83]: > The blog and newsletter are consumption resources, not contact resources. 
> They'd be better referenced in a media sidebar category? Would that media 
> sidebar perhaps also contain the Twitter/Faceback/YouTube references?

I don't like the term "media", but maybe I can warm up to this idea. Maybe a section called "Blogs / Blasts / Socials" that has this information. So our menu would look like this:

image

-Danny

**Pull request closed without merging**